website/content/en/highlights/2024-05-07-0-38-0-upgrade-guide.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
I was thinking we'd include this change in v0.37.1 along with the sink fixes rather than v0.38.0. Alternatively we put this one in v0.38.0, but we add a fix for v0.37.1 that handles reencoding the tags from the object that they are currently being parsed into when `parse_ddtags` is `true`.

Reply:
Indeed, I was also thinking 0.37.1 . 🤔  I guess I'm a little confused where the upgrade guide note for this needs to go then? I dont think we have these upgrade guides for patch releases right? 😕  I might be crossing wires but I thought we'd discussed needing an upgrade note for this change?

Reply:
It is certainly a breaking change. I would just include it in the changelog entry since, as you note, we don't have a precedent for an upgrade guide for patch releases.

Reply:
👍  I'll drop this file